plugins/gnome/extension/notifications.js

```diff
@@ -489,6 +493,18 @@ export const NotificationManager = class extends Signals.EventEmitter {
     }
 
     _showDoNotDisturbButton() {
+        if (DoNotDisturb) {
+            for (const indicator of Main.panel.statusArea.quickSettings._indicators.get_children()) {
+                if (indicator instanceof DoNotDisturb.Indicator) {
+                    for (const qs of indicator.quickSettingsItems) {
+                        qs.remove_style_pseudo_class("insensitive");
+                        qs.buttonMask = St.ButtonMask.ONE;
```

Comment:
I would try setting `qs.reactive = false` - that's a property from `Clutter.Actor`. Setting it adds "insensitive" pseudo class.